runtimes/nodejs/nodejs.md

```diff
@@ -186,10 +186,9 @@ You can (and should) define the `NODE_ENV=production` environment variable as ma
 rely on it for production mode.
 
 
-### Use private repositories with NPM_REGISTRY and NPM_TOKEN
+### Use private repositories with CC_NPM_REGISTRY and NPM_TOKEN
 
-On Clever Cloud, you only need to provide the *token* part. So to register your auth token, you need 
-to add to your application the `NPM_TOKEN` environment variable.
+Since April 2015, npmjs.com allows you to have private repositories. If you want to use a private repository on npmjs.com (the default one), you only need to provide the *token* part. So to register your auth token, you need to add to your application the `NPM_TOKEN` environment variable.
```

Comment:
```suggestion
Since April 2015, npmjs.com allows you to have private repositories. If you want to use a private repository on npmjs.com (the default one), you only need to provide the *token* part. To register your auth token, you need to add to your application the `NPM_TOKEN` environment variable.
```